mtp_noms_ops/apps/security/forms/check.py

```diff
@@ -126,6 +126,40 @@ def get_object_list(self):
         return object_list
 
 
+class AutoAcceptListForm(SecurityForm):
+    """
+    List of AutoAccepts checks.
+    """
```

Comment:
i'm pretty sure you need the ordering field here like in other object-listing forms otherwise no params are sent to the api